src/frontends/tensorflow/src/op/merge.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "helper_ops/merge.hpp"
+
+#include "common_op_table.hpp"
+#include "openvino/frontend/tensorflow/node_context.hpp"
+#include "openvino/opsets/opset11.hpp"
+#include "utils.hpp"
+
+using namespace std;
+using namespace ov;
+using namespace ov::frontend::tensorflow;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_merge_op(const NodeContext& node) {
+    // Merge can have multiple inputs, one is minimum
+    auto node_name = node.get_name();
+    default_op_checks(node, 1, {"Merge"});
+    int input_size = static_cast<int>(node.get_input_size());
+    ov::OutputVector inputs;
+    for (int input_ind = 0; input_ind < input_size; ++input_ind) {
+        inputs.push_back(node.get_input(input_ind));
+    }
+
+    // if Merge node has just one input, there is nothing to merge
+    // return the same input and value_index equal to 0
+    if (inputs.size() == 1) {
+        auto value_index = make_shared<opset11::Constant>(element::i32, Shape{}, 0);
```

Comment:
The current recommendation is to use the concrete op version instead of opset
ov::op::v0::Constant
due to  CVS-91401.

I think this is not reproducible for Frontend lib right now, but we can expect a similar problem in the future.

Reply:
Agree, I seemed to encounter the same issue. Fixed.